data/competencies.yml

```
@@ -710,9 +710,10 @@
   domain: null
 
 - id: 983dd8411-940d-4aa8-9ba6-add04510a7da
-  summary: Is accountable for the delivery of the team
+  summary: Is accountable (individually or jointly with other people) for the delivery of the team
```

Shouldn't stop merging as this is a good improvement, but wondered whether having the brackets at the end reads better?

```suggestion
  summary: Is accountable for the delivery of the team (individually or jointly with other people)
```